src/styles/scss_styles/loops.scss

```diff
@@ -0,0 +1,38 @@
+@import "./variables.scss";
+@import "./mixins.scss";
+@import "./collection.scss";
+
+@for $day_num from 1 through 31 {
+  .calendar__day:nth-child(#{$day_num})::before {
+    content: "#{$day_num}";
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: $font_size;
+  }
+
+  .calendar__day:nth-child(#{$day_num}) {
+    transition-duration: 0.5s;
+    cursor: pointer;
+  }
+
+  .calendar__day:nth-child(#{$day_num}):hover {
+    background-color: pink;
+    cursor: pointer;
+    transform: translateY(-20px);
+  }
```

Comment:
remove this from the @for rule, this can be applied to all days
just use .calendar__day

Reply:
++